Source/Android/app/src/main/AndroidManifest.xml

```diff
@@ -74,6 +78,16 @@
             android:exported="false">
         </provider>
 
+        <provider
+            android:name="android.support.v4.content.FileProvider"
+            android:authorities="${applicationId}.filesprovider"
```

Comment:
I think this breaks debug builds. They have a postfix on the application id. But we may just drop this postfix instead...

Reply:
hmm..probably you are right but I wonder why I didn't see it breaking during the development 🤔

Reply:
Did you receive any bug report regarding this? What make you come back and check it :)